yesand/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class CommentsController < ApplicationController
```

Comment:
Looks like you still have some nested files here.. There's this controller, logs and the tmp directory in yesand/yesand/